c++/binary_search_sidkhuntia.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream> 
+
+#define lli long long int
+#define li long int
+#define ld long double
+#define f(i, n) for (int i; i < n; i++)
+
+using namespace std;
+const lli mod = 1e9 + 7;
+
+int binarySearch(int arr[], int l, int r, int x)
```

Comment:
I would prefer having function declaration above main while the function declaration goes below main as it makes sense to have main above all